flink-python/pyflink/fn_execution/state_impl.py

```diff
@@ -959,7 +1028,8 @@ def clear_cached_iterators(self):
 
     @staticmethod
     def commit_internal_state(internal_state):
-        internal_state.commit()
+        if internal_state is not None:
+            internal_state.commit()
```

Comment:
else return, the following code is also not need to be executed.

Reply:
Yes, although `None` will not affect the execution of the following code, but adding the logic of `else return` can be cleaner